src/clients/abTester.ts

```diff
@@ -8,9 +8,10 @@ const routes = {
   Status: () => `${routes.ABTester}/status`,
 }
 
-export class ABTester extends IODataSource {
-  protected httpClientFactory = forWorkspace
-  protected service = 'ab-tester.vtex'
+export class ABTester extends AppClient {
+  constructor(context: IOContext, options: InstanceOptions) {
+    super('ab-tester.vtex', context, options)
```

Comment:
Is this correct? Maybe the `AppClient` needs to receive the `vendor.name` in the correct order `(vtex.ab-tester`)

Reply:
You are right! Forgot to dis-invert this.